pyproject.toml

```
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "syncstorage-rs"
+version = "0.1.0"
+description = "sync storage server in rust"
```

There's a potential that folk might discover this package while looking for a stand-alone Sync storage server. Let's be exceptionally clear about what this is and how it's being used.

```suggestion
description = "Support package for the Mozilla Sync storage server, written in rust. This is not a 'stand alone' server and is not meant for general use to run your own Sync storage server. Please see [the wiki page](https://github.com/mozilla-services/syncstorage-rs/wiki/overview) for further details."
```